### ISSUE TITLE
unexpected crash app 

### ISSUE THREAD

**Issue by AbbetWang** [electron/electron-packager#1035]
### Preflight Checklist

* [x] I have read the [contribution documentation](https://github.com/electron/electron-packager/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [code of conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* [x] I have searched the issue tracker for a bug that matches the one I want to file, without success.

may be caused by platform.js 
```js
async relativeRename (basePath, oldName, newName) {
    debug(`Renaming ${oldName} to ${newName} in ${basePath}`)
    await fs.rename(path.join(basePath, oldName), path.join(basePath, newName))
  }
```

some output
Renaming Electron Helper (Plugin) to my-app Helper (Plugin) in /var/folders/0f/11s4vtbx2m9757f1708219140000gn/T/electron-packager/darwin-x64/my-app-darwin-x64/Electron.app/Contents/Frameworks/Electron Helper (Plugin).app/Contents/MacOS

the app is 

![image](https://user-images.githubusercontent.com/28219139/62126429-3c086380-b302-11e9-9385-7c6b7f05e7bf.png)

i changed the name

![image](https://user-images.githubusercontent.com/28219139/62126572-930e3880-b302-11e9-8db0-ab33633b1e64.png)

**And it works**

I think it's a bug , but i have no idea to fix it elegantly.

**Comment by AbbetWang** [comment]
> ### Preflight Checklist
> * [x]  I have read the [contribution documentation](https://github.com/electron/electron-packager/blob/master/CONTRIBUTING.md) for this project.
> * [x]  I agree to follow the [code of conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
> * [x]  I have searched the issue tracker for a bug that matches the one I want to file, without success.

i used    
 "electron-packager": "14.0.3",

> 
> may be caused by platform.js
> 
> ```js
> async relativeRename (basePath, oldName, newName) {
>     debug(`Renaming ${oldName} to ${newName} in ${basePath}`)
>     await fs.rename(path.join(basePath, oldName), path.join(basePath, newName))
>   }
> ```
> 
> some output
> Renaming Electron Helper (Plugin) to my-app Helper (Plugin) in /var/folders/0f/11s4vtbx2m9757f1708219140000gn/T/electron-packager/darwin-x64/my-app-darwin-x64/Electron.app/Contents/Frameworks/Electron Helper (Plugin).app/Contents/MacOS
> 
> the app is
> 
> ![image](https://user-images.githubusercontent.com/28219139/62126429-3c086380-b302-11e9-9385-7c6b7f05e7bf.png)
> 
> i changed the name
> 
> ![image](https://user-images.githubusercontent.com/28219139/62126572-930e3880-b302-11e9-8db0-ab33633b1e64.png)
> 
> **And it works**
> 
> I think it's a bug , but i have no idea to fix it elegantly.

I followed this 
#1032

**Comment by AbbetWang** [comment]
why need change helperName
async renameHelperAndExecutable (helperDirectory, originalBasename, newBasename) {
    const originalAppname = `${originalBasename}.app`
    const executableBasePath = path.join(helperDirectory, originalAppname, 'Contents', 'MacOS')
    await this.relativeRename(executableBasePath, originalBasename, newBasename)
    await this.relativeRename(helperDirectory, originalAppname, `${newBasename}.app`)
  }

**Comment by AbbetWang** [comment]
I found that ,because of the Electron Helper (GPU) of electron v6

**Comment by malept** [comment]
Fixed in #1036.
